docs/coding.rst

```diff
@@ -198,6 +198,87 @@ You can also pass template data as keyword arguments:
 
 The variable `latest_version` will be available in the template.
 
+Finding Templates by URL
+------------------------
+
+General Structure
+~~~~~~~~~~~~~~~~~
+
+Bedrock follows the Django app structure and most templates are easy to find by matching URL path segments to folders and files within the correct app.
+
+| URL: ``https://www.mozilla.org/en-US/firefox/features/private-browsing/``
+| Template path:  ``bedrock/bedrock/firefox/templates/firefox/features/private-browsing.html``
+
+To get from URL to template path:
+
+- Ignore ``https://www.mozilla.org`` and the locale path segment ``/en-US``. The next path segment is the app name ``/firefox``.
+- From the root folder of bedrock, find the app's template folder at ``bedrock/{app}/templates/{app}``
+- Match remaining URL path segments (``/features/private-browsing``) to the template folder's structure (``/features/private-browsing.html``)
+
+.. note::
+
+    ``mozorg`` is the app name for the home page and child pages related to Mozilla Corporation (i.e. About, Contact, Diversity).
+
+Whatsnew and Firstrun
+~~~~~~~~~~~~~~~~~~~~~
+
+These pages are specific to Firefox browsers, and only appear when a user updates or installs and runs a Firefox browser for the first time.
+The URL and template depend on what Firefox browser and version are in use.
+
+.. note::
+
+    There may be extra logic in the app's ``views.py`` file to change the template based on locale or geographic location as well.
+
+Firefox (Release)
```

Comment:
Since most of these have qualifies, I find the "Release" in parenthesis unnecessary. I often see it referred to as "Firefox release".

Reply:
Thanks for the review! Will update